### DISCUSSION TITLE
Raspi3, Libreelec and VPN...  unwatchable...

### DISCUSSION THREAD

**Discussion by CharlieTemplar** [retrospect-addon/plugin.video.retrospect#1606] [General]
Live and replay are very stuttery, plays 3 seconds then buffers for 10s and plays another 2-3 seconds, so unwatchable.
What is most likely cause? 
changed the inputstream settings to reduce the resolution which helped but still unwatchable.
weirdly, accidentally streamed only video and without the audio, the video was much better... but without sound, still unwatchable...

Nederland 24 plugin worked flawlessly when is used to work, so think the vpn and raspi are good enough... but willing totweak to get this to work...  any ideas?

**Comment by basrieter** [comment]
Check this: https://github.com/retrospect-addon/plugin.video.retrospect/issues/1601

There is a WideVine issue.

You can also limit the bandwidth of streams via the Retrospect settings.

**Reply by CharlieTemplar** [reply]
reduced from 1080p default to 640 and it helped but still stuttery.  will check the older version of widevine, thanks for the tip.

**Reply by CharlieTemplar** [reply]
Found reference to a library version 4.10.2252.0 working with osmc on raspi, so tried on libreelec.  No more stutters!! :D
used:
```curl https://k.slyguy.xyz/.decryptmodules/widevine/4.10.2252.0-linux-armv7.so --output /storage/.kodi/cdm/libwidevinecdm.so```
The resolution looked terrible, and I remebered, I'd dropped in hte widevine settings to 480p, so set it back to max and restarted, but the stream never got better, no stutters, but 480p on a 4k tv looks terrible.

Do these binaries have different resolution limitations dedpending on which chrome device they were supplied for?

**Reply by basrieter** [reply]
I don't know that, but I know there is an update for InputStream incoming, that should fix this.
